### ISSUE TITLE
issues on evaluation:

### ISSUE THREAD

**Issue by luhc15** [facebookresearch/DetectAndTrack#14]
when I run evaluation code, the  output results:Multiple Object Tracking (MOT) metrics:                               [798/1987]
& MOTA & MOTA & MOTA & MOTA & MOTA & MOTA & MOTA & MOTA & MOTP & Prec & Rec  \\
& Head & Shou & Elb  & Wri  & Hip  & Knee & Ankl & Total& Total& Total& Total\\
& 61.7 & 65.4 & 57.1 & 45.6 & 54.3 &  nan &  nan &  nan & 50.1 &  nan &  nan \\
INFO mpii_eval_engine.py: 289: (60.566788034021066, nan)
INFO mpii_eval_engine.py: 290: ...Done in 173.403025866
 which the MOTA is NAN. what's the reason ?

**Comment by rohitgirdhar** [comment]
Are you using the correct library (numpy etc) versions? The evaluation code is not compatible with older versions.

**Comment by luhc15** [comment]
Version now used :
 numpy = 1.14.5  ; 
pyymal = 3.12; 
matplotlib=2.2.2 ;
 opencv-python = 3.4.0.12;
 setuptools=39.2.0; 
Cython =0.28.2;
 mock = 2.0.0; 
scipy = 1.1.0;
 h5py=2.7.1;
 momery_profile = 0.52.0;
  tqdm=4.23.2; 
shapely=1.6.4.post1; 
pandas=0.21.0;  
CUDA=9.0, GCC=5.4 ; cuDNN = 7.0.5.     
Only cuDNN version is lower than  needed,  I'll update it and check if it works correctly.

**Comment by DonaldKam** [comment]
@luhc15 
Could you plz answer my question if you have test Rohit Girdhar's "DetectandTrack" on yourself?
1.How can i get the 3D ResNet50 pretrained model to test on the 3D ResNet50, as is mentioned in the paper：“Replacing ResNet-50 in Mask R-CNN with Resnet-101 gave us about 2% improvement in MOTA”.
2.As i found “e2e_keypoint_rcnn_R-101-FPN_1x_trainedCOCO.pkl” ,which is imported as Train Weights（in 2D yaml）in the pretrained model file，I wonder whether it is for 2D training or 3D training？If it is for 2D training，where is the 3D pretrained model for ResNet 101？
3.In 3D yamls files，there are options：CONV_BODY and ROI_HEAD. If I used ResNet 101，I found that there is No ROI_HEAD function for ResNet101.
If I choose "ResNet3D.add_ResNet101_conv5_body" for CONV_BODY, what should i do with ROI_HEAD(error occurs when CONV_BODY is "ResNet3D.add_ResNet101_conv5_body" and ROI_HEAD option is null)?
4.As inflating the 2D model to 3D，why are the pretrained models 3D?
